### DISCUSSION TITLE
Add from commandline

### DISCUSSION THREAD

**Discussion by capturts** [jeff-hughes/shellcaster#48] [Ideas]
I love the `shellcaster sync` command. Could there be a `shellcaster add http://example.com/podcast.rss` commandline option?

**Comment by jeff-hughes** [comment]
Yeah, that's a great idea! I'm not looking to turn shellcaster into a full-fledged CLI tool, as there are several other projects with that aim already, but `shellcaster add` seems like a reasonable addition that would be a useful step for just quickly adding a podcast. I'll add it to my list. Thanks!
